src/telegram.ts

```diff
@@ -1,10 +1,28 @@
 import { Telegraf } from "telegraf";
+import HttpsProxyAgent from "https-proxy-agent";
 
-const telegramBot =
+const proxyUrl = process.env.HTTPS_PROXY;
+
+let telegramBot: any ;
```

Comment:
Could you please add the specific type? Like `: Telegraf?` i think?

Reply:
I've now added the type and a description to the README